lib/project_types/script/layers/application/project_dependencies.rb

```diff
@@ -3,6 +3,7 @@ module Layers
     module Application
       class ProjectDependencies
         def self.install(ctx:, task_runner:)
+          return false unless task_runner.has_dependencies?
```

Comment:
Yeah that was my intention. I noticed that the class was returning true or an error, and figured it would make sense to return false when it noop's.

This true/false value is not currently used anywhere directly. 
Instead ends up being received by the surrounding UI element in the CLI, and affects the colour we render.

![](https://screenshot.click/20-19-7sfhl-u0hkbasd.png)

I've added a `true` to `App::CreateScript` after it calls `ProjectDependencies` to fix the UI, while allowing PD to return false to indicate it didn't need to do anything.